public/global-styles.scss

```diff
@@ -11,3 +11,8 @@
   height: 100%;
   width: 100%;
 }
+
+.panel-with-radius {
```

Comment:
Can we reuse the same style from the new Discover?
![image](https://github.com/user-attachments/assets/3d70ca13-7be2-4b5f-a61c-d5f930d5663c)
It should be border-radius: 8px

This radius style should only be applied to the containers within the Visual Editor. The "Manage workflow/Create workflow" page should use the default containers.